### DISCUSSION TITLE
failed to install next 13

### DISCUSSION THREAD

**Discussion by aldiabdumalik** [vercel/next.js#58849] [Help]
### Summary

I installed next 13, but after that it installed 14 by itself

**Comment by icyJoseph** [comment]
What do you mean?

You could do: `yarn add next@13` to install version 13

**Comment by ezeparziale** [comment]
`npm install next@latest ` --> latest version today is 14.0.3

or  

`npm install next@14.0.3`

or 

`npm install next@13.5.6`
